svthalia/concrexit
# Add check if people are still studying

Closes #2883.



### Summary

Add a yearly check 


### How to test

1. Go to create a few members of all types
2. Make their until almost expire
3. Try to renew with all possible ways and see if only the correct one works.
4. See if the honorary member has an until of None

### Still TODO:
(1. make the migration a bit better)
